cmd/lncli/cmd_pay.go

```diff
@@ -303,7 +302,8 @@ func sendPaymentRequest(ctx *cli.Context, req *lnrpc.SendRequest) error {
 		req.LastHopPubkey = lastHop[:]
 	}
 
-	req.CltvLimit = uint32(ctx.Int(cltvLimitFlag.Name))
+	req.CltvLimit = int32(ctx.Int(cltvLimitFlag.Name))
```

Comment:
Is there a reason not using `ctx.Uint32` here? (as `CltvLimit` is `uint32`)

Reply:
`CltvLimit` is `int32` in the proto

Reply:
Sorry, you're right.